module/cli_controller.py

```diff
@@ -46,58 +48,86 @@ def load_tree(self):
         for file in self.command_file_list:
             self.load_commands(file, self.root_node)
 
+    def add_frame_commands(self, root_node):
+        """
+        Adds all commands from the common.frames file
+        """
+        #check if robot already exists, otherwise create it
+        if not 'ROBOT' in root_node:
+            root_node["ROBOT"] = CommandNode("ROBOT")          
+        current_node = root_node["ROBOT"]
+
+        #get class information from common.frames
+        frames = inspect.getmembers(common.frames, lambda member: inspect.isclass(member) and member.__module__ == 'common.frames')
+        #removes files without a desciption
+        commands = [frame for frame in frames if frame[1].DESCRIPTION ]
+        
+        #add all commands to root node
+        for command in commands:
+            target_node = CommandNode(
+                command[0][5:],
+                parameter_list=inspect.getfullargspec(command[1].set_data).annotations,
+                command_info=command[1].DESCRIPTION
+            )
+            print("initialised " + command[0][5:])
```

Comment:
this should probably not be here

Reply:
Wouldn't it be a good indicator if something went wrong?

Reply:
NVM i think it might be better to add these things in a test program